### PR TITLE
102 upgrade workaround

### PR DESCRIPTION
**Pull request by neal-timpe**
https://issues.jboss.org/projects/OSSMDOC/issues/OSSMDOC-10?filter=allissues

@tvieira @yxun Can you take a look and let me know if you have any feedback? Does anyone else need to look at this also?

### CODE REVIEW

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (yxun) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: @neal-timpe line 35 It's better to replace "Update OSSM after all pods are running in the SMCP namespace." with "Upgrade OSSM operator and SMCP to 1.0.2. After all pods are running in the SMCP namespce, "

**Review by yxun — APPROVED** [review]

**Review by tvieira — APPROVED** [review]
LGTM

**Comment by neal-timpe** [comment]
@openshift/team-documentation Can you review this PR? It will be for 4.2 and 4.3.
Also, I have a go/no-go meeting later today. So after it's reviewed, can you wait for a signal before we merge?

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Throughout:

s/OpenShift/{product-title}/

Unless there was a reason to not do this in Service Mesh docs specifically.

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Markup the replaceables, but also use underscores instead of spaces, e.g., `<smcp_namespace>`.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Comment (adellape) [reply service_mesh/service_mesh_install/updating-ossm.adoc]: Also consider:

s/patch/`oc patch`/

Comment (adellape) [reply service_mesh/service_mesh_install/updating-ossm.adoc]: s/names./names:/

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Consider chunking it up with line breaks[1] so it's more easily readable (even if the spec blob stays on its own line)? This will also allow for more targeted callouts (for example, the "e.g. basic-install" could then be a callout). E.g.:

```
----
$ oc patch -n <smcp_namespace> smcp <smcp_name> \// <1>
    --type=merge -p \
    '{"spec": {"istio": {"global": {"defaultResources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}},"proxy": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"},"limits":{}}},"defaultPodDisruptionBudget": {"enabled": false}},"security": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"galley": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"pilot": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"mixer": {"telemetry": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"gateways": {"istio-egressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}},"istio-ingressgateway": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}},"prometheus": {"resources": {"requests": {"cpu": "10m","memory": "128Mi"}}}}}}'
----
<1> For example, `basic-install`.
```

Re: `\// <1>`, usually a simple `<1>` would be fine, but since we don't want whitespace after the `\` line break, this makes it more copy-paste friendly. [2]

[1] https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks
[2] https://asciidoctor-docs.netlify.com/asciidoc/1.5/source/callout/

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Throughout, init cap "Pod".

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Markup `cluster-admin` (a role name).

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Throughout, init cap "Operator".

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization

Comment (adellape) [reply service_mesh/service_mesh_install/updating-ossm.adoc]: s/namespace./namespace:/

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [reply service_mesh/service_mesh_install/updating-ossm.adoc]: s/such as `kubeadmin`/, such as `kubeadmin`,/

Comment (adellape) [reply service_mesh/service_mesh_install/updating-ossm.adoc]: s/Daemonset/DaemonSet/

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: s/deployment./deployment:/

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: s/console/web console/

Comment (adellape) [reply service_mesh/service_mesh_install/updating-ossm.adoc]: Maybe:

s/with/using/

?

File: release_notes/ocp-4-2-release-notes.adoc
Comment (adellape) [inline release_notes/ocp-4-2-release-notes.adoc]: Add the anchor:

```
xref:../service_mesh/service_mesh_install/updating-ossm.adoc#updating-ossm[Updating OpenShift Service Mesh from version 1.0.1 to 1.0.2].
````

**Review by adellape — CHANGES_REQUESTED** [review]

File: service_mesh/service_mesh_install/updating-ossm.adoc
Comment (adellape) [inline service_mesh/service_mesh_install/updating-ossm.adoc]: Oops, sorry, I thought I entered a comment here. Same deal with the replaceables (lowercase w/ underscores), and consider line-breaking:

```
----
$ oc patch -n <data_plane_namespace> deployment/<deployment_name> -p \
    '{"spec":{"template":{"metadata":{"annotations":{"kubectl.kubernetes.io/restartedAt": "'`date -Iseconds`'"}}}}}'
----
```

**Review by adellape — APPROVED** [review]
Changes LGTM! One comment I forgot to say last time though.

**Pull request merged**
